### PR TITLE
Next docs update

### PR DESCRIPTION
**Pull request by icode247**

### CODE REVIEW

File: docs/guides/nextjs/add-existing-app/index.mdx [hunk 1]
```diff
@@ -2,241 +2,243 @@
...
+  <CodeBlock language="shell">{createNextAppOutput}</CodeBlock>
 </TerminalWindow>
-```
 
 This is a simple command to quickly start building a new Next.js application based off a template.
 It may ask you a project name to download the project resources. We choose `my-nextjs-blog` as our
 project name.
 
-Change to `my-nextjs-blog` directory and install the dependencies:
+Change to `my-nextjs-blog` directory and install the the Tigris SDK:
```
Comment (leggetter) [inline docs/guides/nextjs/add-existing-app/index.mdx:39]: double "the"

File: docs/guides/nextjs/add-existing-app/index.mdx [hunk 2]
```diff
@@ -2,241 +2,243 @@
...
 
-```mdx-code-block
-<TerminalWindow>
-<CodeBlock language="shell">{loginOutput}</CodeBlock>
-</TerminalWindow>
-```
+![The create a project dialog within the Tigris console web application](/img/screenshots/create-new-project-dialog.jpg)
 
-### Add Tigris Cloud Credentials
+We will be using `MyNextjsBlog` as the project name.
```
Comment (leggetter) [inline docs/guides/nextjs/add-existing-app/index.mdx:54]: I'm not quite sure how to explain why but I think we should be more instructional here e.g.

Use `MyNextjsBlog` as the project name.

File: docs/guides/nextjs/add-existing-app/index.mdx [hunk 3]
```diff
@@ -2,241 +2,243 @@
...
-Now, create a `.env.development.local` file in your project root directory and add the application credentials generated above to it:
+We will use the project name, clientId and clientSecret from the previous step
 
-```dotenv title=.env.development.local
+```dotenv title=.env.local
 TIGRIS_URI=api.preview.tigrisdata.cloud
-TIGRIS_CLIENT_ID=client_id_here
-TIGRIS_CLIENT_SECRET=client_secret_here
+TIGRIS_PROJECT=MyNextjsBlog
+TIGRIS_CLIENT_ID=ftSUj9B5czFW79s9M6YUkxKE3H4WeRyY
```
Comment (leggetter) [inline docs/guides/nextjs/add-existing-app/index.mdx:76]: Question about projects: isn't a client ID tied to a project? If so, why do we also need to provide a project name?

File: docs/guides/nextjs/add-existing-app/index.mdx [hunk 4]
```diff
@@ -245,45 +247,41 @@ type Response = {
...
-Here we used a simple `findMany()` construct to read documents from a Tigris collection and returned
-them as json.
+Here we used a simple `findMany()` function to read documents from a Tigris
+collection and returned them as json.
 
-#### ⌲ POST `/api/comments` to add new comment to the collection.
+#### ⌲ POST `/api/comments` to add a new blog data.
 
-Now, create a `POST` method in `pages/api/comments/index.ts` create a new comment with the code:
+Now, create a `POST` handler in the same file `pages/api/comments/index.ts` where
```
Comment (leggetter) [inline docs/guides/nextjs/add-existing-app/index.mdx:265]: I feel there's some inconsistency with the use of commas after "Now" and "Next".

Embarrassingly, I had to perform a search to find out these are called [sequencers](https://www.engvid.com/english-writing-sequencing-first-next-finally/) 😊

**Review by leggetter — COMMENTED** [review]
I know this has been merged but I wanted to give it a review anyway, as it's a good learning experience for me.

File: docs/guides/nextjs/add-existing-app/index.mdx (hunk 3 above)
Comment (ovaistariq) [reply docs/guides/nextjs/add-existing-app/index.mdx:76]: We looked at Firebase and Supabase; they all have a way for developers to quickly identify from their application code what project their application is using. If we only have clientID then this information is hidden from them.

File: docs/guides/nextjs/add-existing-app/index.mdx (hunk 4 above)
Comment (ovaistariq) [reply docs/guides/nextjs/add-existing-app/index.mdx:265]: I wish I could use Grammarly in my code IDE 😄

**Comment by ovaistariq** [comment]
@leggetter thanks for the review comments, I have addressed them here: https://github.com/tigrisdata/tigris-docs/pull/268

File: docs/guides/nextjs/add-existing-app/index.mdx (hunk 4 above)
Comment (icode247) [reply docs/guides/nextjs/add-existing-app/index.mdx:265]: You can copy the markdown to Grammarly and copy them back to your IDE. It works too

Comment (leggetter) [reply docs/guides/nextjs/add-existing-app/index.mdx:265]: There is a VSCode plugin https://marketplace.visualstudio.com/items?itemName=znck.grammarly

Comment (ovaistariq) [reply docs/guides/nextjs/add-existing-app/index.mdx:265]: Unfortunately or fortunately I am a JetBrains users

**Pull request merged**